Tensile/KernelWriterAssembly.py

```diff
@@ -6235,7 +6246,10 @@ def endSummation(self, kernel, label = None, isOptNLL = False):
       #kStr += "s_nop %u\n" % instCycles
       kStr += self.MapAcctoArchRegs(kernel,option=0, isOptNLL=isOptNLL)
       if kernel["MIArchVgpr"] or kernel["StoreCInUnroll"]:
-        kStr += self.MulMIoutAlphaToArch(kernel)
+        if kernel["_GlobalAccumulation"] == 'MultipleBuffer':
```

Comment:
I added extra vgpr allocation for MulMIoutAlphaToArch with complex case.
(from line 1519 to line 1523 before change).
If MoveMIoutToArch is called, the extra vgpr allocation is unnecessary.
Would you please add " and  kernel["_GlobalAccumulation"] != 'MultipleBuffer'" at line 1519 (before change) to avoid unnecessary extra vgpr allocation?